tests/copy-paste.js

```diff
@@ -685,13 +685,13 @@ define([
             paste([
                 ["id", "type", "labelItext:en-default", "itemsetData"],
                 ["/select", "SelectDynamic", "select",
-                 '[{"instance":null,"nodeset":"/items","labelRef":"@name","valueRef":"@id"}]'],
+                 '[{"instance":null,"nodeset":"/items","labelRef":"name","valueRef":"@id"}]'],
```

Comment:
Not sure if this was a legitimate change to make. When I manually test copy & paste, it copies over the value & label correctly, but the test was failing to copy the labelRef correctly.